src/main/resources/external-user-mgmt-api.yaml

```diff
@@ -207,6 +207,42 @@ paths:
           description: Bearer token is expired, or it doesn't have the 'manage-user' scope, or the target user's roles are not included in the current user's assignable roles.
         400:
           description: Required parameters are missing or invalid.
+        404:
+          description: The user could not be found.
+      security:
+        - auth:
+            - manage-user
+
+  /users/sso/{userId}:
```

Comment:
Doesn't match the description in the ticket (sso should be at end):
`DELETE /api/v1/users/{userId}/sso`